tests/unit/ops/fp_quantizer/test_fp8_gemm.py

```diff
@@ -25,7 +26,11 @@
 def test_fp_quant(dtype, q_bits, M):
     device_name = get_accelerator().device_name()
     quantization_group_size = 128
-    fpq = FP_Quantize(group_size=quantization_group_size)
+
+    quant_config = QuantizationConfig()
```

Comment:
Will the following work?
```suggestion
    quant_config = QuantizationConfig(group_size = quantization_group_size, q_dtype = FPQuantizerBuilder.get_default_quant_dtype())
```

Reply:
Done